crates/apub/src/activities/mod.rs

```diff
@@ -97,7 +97,9 @@ pub(crate) async fn verify_person_in_community(
 ) -> Result<(), LemmyError> {
   let person = person_id.dereference(context).await?;
   if person.banned {
-    return Err(LemmyErrorType::PersonIsBannedFromSite)?;
+    return Err(LemmyErrorType::PersonIsBannedFromSite(
+      person.name.to_owned(),
```

Comment:
person.name is not unique, there can be multiple users with the same name on different instances. Better to use `person.actor_id`

Reply:
I was considering this might be better. Good catch